README.md

```diff
@@ -65,6 +69,7 @@ jobs:
           docker_name: 'owasp/zap2docker-stable'
           target: 'https://www.example.com'
           rules_file_name: '.zap/rules.tsv'
+          cmdOptions: '- a'
```

Comment:
Sould the space between the `-` and the `a` really be there?

Reply:
Fixed.